src/components/custom/signup.js

```diff
@@ -129,51 +117,51 @@ class Signup extends Component {
             const response = JSON.parse(msg.data)
             if (response.error) {
                 binary_socket.close()
-                this.setState({
-                    is_submitting: false,
-                    submit_status: 'error',
-                    submit_error_msg: response.error.message,
-                })
-                this.handleValidation(email)
+                setSubmitting(false)
```

Comment:
Since `setSubmitting(false)` is called in either conditions, we probably can move this outside the condition

Reply:
Resolved